src/locale/locales/ro/messages.po

```diff
@@ -311,17 +311,17 @@ msgstr "{profileName} s-a alăturat Bluesky acum {0}"
 
 #: src/components/NewskieDialog.tsx:111
 msgid "{profileName} joined Bluesky using a starter pack {0} ago"
-msgstr "{profileName} s-a alăturat Bluesky folosind un pachet de start acum {0}"
+msgstr "{profileName} s-a alăturat Bluesky folosind un pachet de început acum {0}"
 
-#: src/screens/StarterPack/Wizard/index.tsx:475
+#: src/screens/StarterPack/Wizard/index.tsx:472
 msgctxt "profiles"
 msgid "<0>{0}, </0><1>{1}, </1>and {2, plural, one {# other} other {# others}} are included in your starter pack"
-msgstr "<0>{0}, </0><1>{1}, </1>și {2, plural, one {un alt utilizator} few {alți # utilizatori} other {alți # de utilizatori}} sunt incluși în pachetul tău de start"
+msgstr ""
```

Comment:
This is not a suppressed translation but it seems the translation has been removed